app/not-found.tsx

```diff
@@ -1,13 +1,24 @@
-import Link from 'next/link';
+'use client';
+
+import NotFoundAnimation from '../public/lottiefiles/404.json';
+import { Player } from '@lottiefiles/react-lottie-player';
 
 export default function NotFound() {
+  
   return (
-    <div className="flex flex-col m-auto">
-      <h2 className="font-bold flex flex-col m-auto">Error 404</h2>
-      <p>Oops! This page was not found</p>
-      <button className="border-2">
-        <Link href="/">Return Home</Link>
-      </button>
+    <div>
```

Comment:
`Page Not Found` and `We couldn't find the page you're looking for' seems duplicated. Maybe remove one of them?

Reply:
Also the same comment about centering the animation.